packages/plugin-auth-sms/src/client/SigninPage.tsx

```
@@ -3,8 +3,18 @@ import { SchemaComponent } from '@tachybase/client';
 import { Authenticator, useSignIn } from '@tachybase/module-auth/client';
 import { ISchema } from '@tachybase/schema';
 
+import { Checkbox } from 'antd';
+
 import VerificationCode from './VerificationCode';
 
+const HtmlAgreementCheckbox = ({ value, onChange, htmlCode = '' }: any) => {
+  return (
+    <Checkbox checked={value} onChange={(e) => onChange(e.target.checked)}>
+      <span dangerouslySetInnerHTML={{ __html: htmlCode }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)